static/js/settings_account.js

```diff
@@ -871,4 +872,31 @@ export function set_up() {
             $("#account-settings .privacy-setting-status").expectOne(),
         );
     });
+
+    $("#deactivate_realm_personal_button").on("click", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        function do_deactivate_realm() {
+            channel.post({
+                url: "/json/realm/deactivate",
+                error(xhr) {
+                    ui_report.error(
+                        $t_html({defaultMessage: "Failed"}),
+                        xhr,
+                        $("#admin-realm-deactivation-status").expectOne(),
```

Comment:
This error element is present in the `Organization profile` page. I think we can instead show the error in the modal itself for both the cases like we do for other places where we use modal. For organization profile page as well, the error is shown at the top and we need to scroll back up to check whether there is an error or not, so I think showing error in modal is better.

Reply:
Also, we can probably avoid writing the same code twice by adding same class to buttons in both organization and personal section and then writing single handler for it.

Reply:
> This error element is present in the `Organization profile` page. I think we can instead show the error in the modal itself for both the cases like we do for other places where we use modal. For organization profile page as well, the error is shown at the top and we need to scroll back up to check whether there is an error or not, so I think showing error in modal is better.

Working on it 👍